ADAL/src/utils/ios/ADKeychainUtil.m

```diff
@@ -53,15 +53,16 @@ + (NSString*)retrieveTeamIDFromKeychain:(ADAuthenticationError * __autoreleasing
     NSDictionary *query = @{ (id)kSecClass : (id)kSecClassGenericPassword,
                              (id)kSecAttrAccount : @"teamIDHint",
```

Comment:
Changed the PR description to better reflect the issue. The Intune SDK DLP changes the kSecAttrAccessible attribute from kSecAttrAccessibleAlways to kSecAttrAccessibleAlwaysThisDeviceOnly in the original SecItemAdd call. Since kSecAttrAccessibleAlways is included in the SecItemCopyMatching query, applications not managed by the Intune SDK will fail to find the keychain item and will attempt to add it, resulting in a -25299 error. The kSecAttrAccessible attribute is not needed in the SecItemCopyMatching query. It's only needed in the SecItemAdd query.